File: frpfun/src/main/java/com/wxfactory/kcps/frpfun/util/Const.java
package com.wxfactory.kcps.frpfun.util;

import cn.hutool.setting.dialect.Props;

import java.nio.charset.StandardCharsets;
import java.util.Properties;

/**
 * @author xhvvvv
 * @time 2024/3/16
 */
public class Const {
    public static Properties ppties ;
    static {
        ppties = Props.getProp("remindText.properties", StandardCharsets.UTF_8).toProperties();
    }
    
}

File: shared/src/commonMain/kotlin/com/wxfactory/kcps/common/util/Const.kt
package com.wxfactory.kcps.common.util

import cn.hutool.setting.dialect.Props
import java.nio.charset.StandardCharsets
import java.util.*

val i18N : Properties by lazy {
     Props.getProp("in18-Zn.properties", StandardCharsets.UTF_8).toProperties()
}
val someStatic : Properties by lazy {
     Props.getProp("pro.properties", StandardCharsets.UTF_8).toProperties()
//     Props.getProp("test.properties", StandardCharsets.UTF_8).toProperties()
}